### ISSUE TITLE
How can I visualize my label file??

### ISSUE THREAD

**Issue by tianyuluan** [AlexeyAB/darknet#6234]
Now ,I want to know how about the model that i trained in my custom dataset，，so i want to make a comparison on  the same picture（predicted and annotated） ,,is there any way to show the label on the picture by darknet frame.
Thank you very much!!!!!

**Comment by kadhirumasankar** [comment]
Just to confirm, do you want to test your model on a training image?

**Comment by KacperPaszkowski** [comment]
I don't think that darknet have this ability to visualize IoU. You need to look online for such a script or write it yourself.
